### PR TITLE
Crop the SAR object before performing operations

### PR DESCRIPTION
**Pull request by paulina-t**

### CODE REVIEW

File: sar.py [hunk 1]
```diff
@@ -84,22 +105,25 @@ def sar_params(sar_fn, normalize=True, vv=True):
...
-        Radar look incidence angle at given location.
+        Radar look incidence angle of the cropped object.
     az : float
-        Radar look azimuth angle at given location.
-    lon : float
-        Geographical longitude in degrees.
-    lat : float
-        Geographical latitude in degrees.
+        Radar look azimuth of the cropped object.
+    lon : array of floats
```
Comment (setmar) [inline sar.py:113]: Should be lon_grid

File: sar.py [hunk 2]
```diff
@@ -84,22 +105,25 @@ def sar_params(sar_fn, normalize=True, vv=True):
...
+        Geographical longitudes in degrees of the cropped object.
+    lat : array of floats
```
Comment (setmar) [inline sar.py:115]: Should be lat_grid

**Review by setmar — COMMENTED** [review]
Typo in doc

File: sar.py (hunk 2 above)
Comment (paulina-t) [reply sar.py:115]: Fixed that in the latest version :)

**Pull request merged**